scripts/deploy_website.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+target=/tmp/cobrapy-website
+
+git clone "https://github.com/opencobra/cobrapy-website.git" "${target}"
+cp -f "${TRAVIS_BUILD_DIR}"/release-notes/[0-9]*.md "${target}/content/releases/"
```

Comment:
This won't work since the hugo md also need a specific TOML header. For instance you have to fill in the type of release (major, minor, patch) and some more stuff. There is a template in archetypes that you could use.

Reply:
Fair enough, I'll look into that.

Reply:
Maybe we rather make that script Python? Might be easier to assemble the header. Major/minor/patch can be decided based on the version number (x.0.0/x.y.0/rest). I could also contribute a part of you want to :)

Reply:
Added a script as you suggested. Locally it does the job.